tests/python/unittest/test_numpy_op.py

```diff
@@ -2871,6 +2871,118 @@ def check_svd(UT, L, V, data_np):
                 check_svd(UT, L, V, data_np)
 
 
+@with_seed()
```

Comment:
This test is not related, please remove it.

Reply:
:+1:

Reply:
So while we cherrypick the PR can we just select part of that PR? only thing needed from that PR is the change in `src/operator/nn/softmax-inl.h`

Reply:
@ChaiBapchya Actually we also need the mixed precision true_divide... d2l-en depends on that now.